spec/lib/schedule_spec.rb

```diff
@@ -21,10 +22,10 @@
       end
     end
 
-    it "rejects unbookable dates" do
+    it "doesn't offer unbookable dates" do
       subject.dates(start_date).each do |date|
-        expect(date).not_to eq(Date.parse('2014-12-25'))
-        expect(date).not_to eq(Date.parse('2014-12-26'))
+        expect(date).not_to eq(Date.new(2014, 12, 25))
```

Comment:
I don't mind either way, I just changed this to match the rest of the file because this was the only use of parse.